src/views/OfferInfoPage/components/CheckOfferPage.tsx

```diff
@@ -17,8 +21,15 @@ const CheckOffer = () => {
     navigate(-1);
   };
 
+  //모달창 열림
+  const [isModal, setIsModal] = useState(false);
+  const handleModalOpen = () => {
+    setIsModal(true);
+  };
+
   return (
-    <>
+    <ScrollToTop>
```

Comment:
```suggestion
    <>
      <ScrollToTop/>
```
감싸줄 필요 없이 이렇게 내부에 셀프 클로징 컴포넌트로 넣어줘도 됩니당!